src/transformers/models/tvlt/modeling_tvlt.py

```diff
@@ -0,0 +1,1380 @@
+
+    def _set_gradient_checkpointing(self, module, value=False):
+        if isinstance(module, TvltEncoder):
+            module.gradient_checkpointing = value
+
+
+TVLT_START_DOCSTRING = r"""
+    This model is a PyTorch [torch.nn.Module](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) subclass. Use it
+    as a regular PyTorch Module and refer to the PyTorch documentation for all matter related to general usage and
+    behavior.
+
+    Parameters:
+        config ([`TvltConfig`]): Model configuration class with all the parameters of the model.
+            Initializing with a config file does not load the weights associated with the model, only the
+            configuration. Check out the [`~PreTrainedModel.from_pretrained`] method to load the model weights.
+"""
+
+TVLT_INPUTS_DOCSTRING = r"""
+    Args:
+        pixel_values (`torch.FloatTensor` of shape `(batch_size, num_frames, num_channels, height, width)`):
+            Pixel values. Pixel values can be obtained using [`TvltPixelFeatureExtractor`]. See
+            [`TvltFeatureExtractor.__call__`] for details.
+
+        audio_values (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
+            Audio values. Audio values can be obtained using [`TvltAudioFeatureExtractor`]. See
+            [`TvltFeatureExtractor.__call__`] for details.
+
+        pixel_masks (`torch.FloatTensor` of shape `(batch_size, num_pixel_patches)`):
+            Pixel masks. Pixel masks can be obtained using [`TvltPixelFeatureExtractor`]. See
+            [`TvltFeatureExtractor.__call__`] for details.
+
+        audio_masks (`torch.FloatTensor` of shape `(batch_size, num_audio_patches)`):
+            Audio masks. Audio masks can be obtained using [`TvltAudioFeatureExtractor`]. See
+            [`TvltFeatureExtractor.__call__`] for details.
+
+        pixel_masking_noise (`torch.FloatTensor` of shape `(batch_size, num_pixel_patches)`):
+            Noises for pixel masks for MAE reconstruction. Pixel masks can be obtained using
+            [`TvltPixelFeatureExtractor`]. See [`TvltFeatureExtractor.__call__`] for details.
+
+        pixel_values_mixed (`torch.FloatTensor` of shape `(batch_size, num_frames, num_channels, height, width)`):
+            Pixel masks of pixel_values_mixed. Pixel values mixed can be obtained using [`TvltPixelFeatureExtractor`].
+            See [`TvltPixelFeatureExtractor.__call__`] for details.
+
+        pixel_masks_mixed (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
+            Pixel values that mixe positive and negative samples in Tvlt vision-audio matching. Audio values can be
+            obtained using [`TvltPixelFeatureExtractor`]. See [`TvltPixelFeatureExtractor.__call__`] for details.
+
+        labels (`torch.FloatTensor` of shape `(batch_size, num_labls)`):
+            Number of labels for pretrainings. For example, whether vision and audio match in pretraining.
```

Comment:
Not sure this should be in the general inputs docstring, since this is only required for TvltForPreTraining I suppose